tests/virt/node/hotplug/test_cpu_memory_hotplug.py

```
@@ -134,8 +134,8 @@ def test_hotplug_memory(self, hotplugged_sockets_memory_guest, hotplugged_vm):
 
     @pytest.mark.dependency(depends=[f"{TESTS_CLASS_NAME}::hotplug_memory"])
     @pytest.mark.polarion("CNV-10677")
-    def test_migrate_snapshot_hotplugged_vm(self, hotplug_vm_snapshot, hotplugged_vm):
-        migrate_vm_and_verify(vm=hotplugged_vm, check_ssh_connectivity=True)
+    def test_migrate_snapshot_hotplugged_vm(self, admin_client, hotplug_vm_snapshot, hotplugged_vm):
```

Ruff (0.15.14)

[warning] Unused method argument: `hotplug_vm_snapshot`

(ARG002)